code/modules/mob/living/simple_animal/hostile/gremlin.dm

```diff
@@ -1,5 +1,5 @@
 //Gremlins
-//Small monsters that don't usually attack humans or other animals. Instead they mess with electronics, computers and machinery
+//Small monsters that don't attack humans or other animals. Instead they mess with electronics, computers and machinery
```

Comment:
Unnecessary. Please get into the habit of opening a file with one set of changes in mind, and then sticking to that. There's no good reason to randomly touch things as you go, and it's nice to keep it as minimal as possible.